components/tls/pkg/tls/util.go

```diff
@@ -63,3 +67,13 @@ func GetSASLMechanismFromEnv(prefix string) string {
 	}
 	return val
 }
+
+func GetEndpointIdentificationMechanismFromEnv(prefix string) string {
```

Comment:
Is there any other instance where the `prefix` is different to `KAFKA_CLIENT`?

Reply:
It may be `KAFKA_BROKER` I believe - this is following logic of other helpers.